tests/shallowMount.spec.ts

```diff
@@ -2,6 +2,7 @@ import { defineAsyncComponent, defineComponent } from 'vue'
 import { mount, shallowMount, VueWrapper } from '../src'
 import ComponentWithChildren from './components/ComponentWithChildren.vue'
 import ScriptSetupWithChildren from './components/ScriptSetupWithChildren.vue'
+import ScriptSetupWithBuiltinComponent from './components/ScriptSetupWithBuiltinComponent.vue'
```

Comment:
nit: I would rename to `DynamicComponentWithComputedProperty` or something like that.